common/src/main/java/com/wynntils/wynn/model/SpellModel.java

```diff
@@ -21,8 +20,11 @@
 import net.minecraftforge.eventbus.api.SubscribeEvent;
 
 public class SpellModel extends Model {
-    private static final Pattern SPELL_TITLE_PATTERN =
-            StringUtils.compileCCRegex("§([LR]|Right|Left)§-§([LR?]|Right|Left)§-§([LR?]|Right|Left)§");
+    // If you modify please test with
```

Comment:
```suggestion
    // If you modify please test on link below
```

Reply:
![image](https://user-images.githubusercontent.com/57310593/213775208-3e3bf5e3-ba7e-4551-86cf-fa6104186c29.png)

I'm not sure why your copy is missing the latest commit, this was fixed in a52aab7